parser/src/parser/context.rs

```diff
@@ -75,12 +91,15 @@ impl ParserContext {
     }
 
     ///
-    /// Returns a reference to the next token or error if it does not exist.
+    /// Returns a reference to the next SpannedToken or error if it does not exist.
```

Comment:
Naming nit: the previous method returns the "next next token" but is labeled `peek_next`.
Should the previous method's description be renamed to "next token"?

If so, then we should update the description here to:
```suggestion
    /// Returns a reference to the current SpannedToken or error if it does not exist.
```

Reply:
Since fixed the documentation here. Peek looks at the current token, peek_next looks at the next token.